did/plugins/trac.py

```diff
@@ -165,8 +165,8 @@ class TracUpdated(TracCommon):
 
     def fetch(self):
         log.info("Searching for tickets updated by {0}".format(self.user))
-        query = "time=..{2}&modified={1}..".format(
-            self.user.login, self.options.since, self.options.until)
+        query = "time=..{1}&modified={0}..".format(
+            self.options.since, self.options.until)
```

Comment:
Now that you're at it, why not go with the following?

```Python
query = f"time..{self.options.until}&modified={self.options.since}.."
```

How did the 2-1 thing get in here in the first place? Isn't there a test to check this?

Reply:
We could convert to format strings but  `format()` is used over 20 times across the file so let's keep that for a separate pull request to keep the code consistent.

> How did the 2-1 thing get in here in the first place? Isn't there a test to check this?

Well,, probably just a copy & paste of the other stat. Trac is not used much recently so I guess the plugin will eventually die. Test improvements are always welcome ;)

Reply:
Haha, not from me ;)